tests/integ/modin/test_numpy.py

```diff
@@ -98,6 +98,7 @@ def test_np_ufunc_arithmetic_operators(np_ufunc):
         assert_array_equal(np.array(snow_result), np.array(pandas_result))
 
 
+@sql_count_checker(query_count=2)
```

Comment:
FYI: I was curious about why we would have any queries at all. It turns out the numpy error message here will print representations of both the inputs, so we execute the same query twice to print `df["A"]`.

Reply:
So this is actually coming from numpy error handling; and it is called from a C function when the __array_ufunc__ lookup ultimately failed. I don't think we necessarily need to fix this because:
1. It occurs on an error condition only
2. Its not in our code exactly; it's numpy formatting errors.

This was a good question though.